pandas/io/common.py

```diff
@@ -150,6 +150,23 @@ def urlopen(*args, **kwargs):
     return urllib.request.urlopen(*args, **kwargs)
 
 
+def get_fs_for_path(filepath):
+    """
+    Get appropriate filesystem given a filepath.
+    Support s3fs, gcs and local disk fs
```

Comment:
can you make this a full doc-string Paramateres / Returns

Reply:
Sure done :)